cli/sawtooth_cli/block.py

```diff
@@ -94,50 +91,52 @@ def print_yaml(data):
         keys = ('num', 'block_id', 'batches', 'txns', 'signer')
         headers = (k.upper() if k != 'batches' else 'BATS' for k in keys)
 
-        def get_block_data(block):
+        def get_data(block):
             batches = block.get('batches', [])
-            txn_count = reduce(
-                lambda t, b: t + len(b.get('transactions', [])),
-                batches,
-                0)
-
+            txns = [t for b in batches for t in b.get('transactions', [])]
```

Comment:
I think I tried something like that, but it didn't work. Must have had it slightly off. Damn list comprehensions...